runners.go

```diff
@@ -84,6 +84,7 @@ type RunnerDetails struct {
 		Name   string `json:"name"`
 		WebURL string `json:"web_url"`
 	} `json:"groups"`
+	MaintenanceNote string `json:"maintenance_note"`
```

Comment:
I try to follow the order in the docs for easy lookup, so that would place this field in between `CantactedAt` and `Name`., can you update that?

Reply:
Will do when I'm back at my personal laptop (late tomorrow PT)!

Reply:
Done!